src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -42,6 +47,11 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
+
```

Comment:
> Complex is better than complicated...
`return getNumberOfOccurrences(array, value) > 0`

Reply:
I'm going to slow down. I realize I'm working harder not smarter. Thanks for the feedback. Keep it coming:)